cloud-config/aws/README.md

```diff
@@ -0,0 +1,156 @@
+This Cloud-Config setup is intended to be used with the Ubuntu Server 18.04 LTS AMI in AWS. You can search for the latest AMI id for your AWS zone at https://cloud-images.ubuntu.com/locator/ec2. Search for `18.04 amd64 hvm:ebs-ssd`
+
+**To use this setup, you must able to link a domain name to EC2 machine's IP address. If you don't own a domain, services such as [FreeDNS](https://freedns.afraid.org) offer free sub-domains under a range of domains.**
+
+## Prerequisites
+
+**Warning**: Make sure at all time that you have selected the availability zone where you want to perform your actions. You can choose the availability zone using the dropdown menu at the top-right corner of the AWS console website.
+
+### Create a VPC
+
+- Go to the [VPC Dashboard](https://console.aws.amazon.com/vpc/home#dashboard)
+  
+  Click on `Launch VPC Wizard`.
+
+- Configure the new VPC
+
+  Follow the wizard for the `VPC with a Single Public Subnet` configuration.
+  
+  Give your VPC a name setting a value in the `VPC Name` field.
+  
+  All the defaults are OK. 
+  
+  Click on `Create VPC`.
+  
+### Create a security group
+
+- Go to the [VPC - Security groups panel](https://console.aws.amazon.com/vpc/home#SecurityGroups:sort=groupId)
+
+  Click on `Create security group`.
+
+- Create the new security group
+
+  Set a name and a description for your new security group.
+  
+  Select the VPC you've created in the previous step.
+  
+  Click on `create`.
+- Set the inbound rules in the security group
+
+  Click on the newly created security group from the list.
+
+  Click on the `Inbound rules` tab, and click on the `Edit rules` button.
+
+  Add the following rules:
+  
+  | Type | Source |
+  | --- | --- |
+  | SSH | Anywhere |
+  | HTTP | Anywhere |
+  | HTTPS | Anywhere |
+  
+  Click on `Save rules`.
+
+## Create an IAM role
+
+- Go to the [IAM - Roles panel](https://console.aws.amazon.com/iam/home#/roles)
+
+
+  Click on `Create role`.
+- Configure the new role
+
+  Make sure that the `AWS service` box is selected, and click on the `EC2` link.
+  
+  Click on `Next: Permissions`.
+- Add policies to the role
+
+  Search for `AmazonEC2ReadOnlyAccess`, and select it.
+  
+  Click on `Next: Tags`. (nothing to do here)
+  
+  Click on `Next: Review`.
+- Give the new IAM role a name
+
+  Click on `Create role`.
+
+## Create the EC2 machine
+
+- Go to the [EC2 Dashboard](https://console.aws.amazon.com/ec2/v2/home#Home:)
+  
+  Click on `Launch instance`
+- Search for the `Ubuntu Server 18.04` AMI
```

Comment:
If you search for `Ubuntu Server 18.04 LTS ` then you'll only get one option, which is what we want.